ChaosMod/Effects/db/Player/PlayerHasGravity.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+	Effect by Last0xygen
+*/
+
+#include <stdafx.h>
+
+static void OnTick()
+{
+	Ped playerPed = PLAYER_PED_ID();
+	std::vector<Entity> entities;
+	//make player not die instantly
+	SET_PLAYER_INVINCIBLE(playerPed, true);
```

Comment:
First arg requires the player (`PLAYER_ID()`) to actually work, not the player ped